### PR TITLE
Translation: pl_PL.po 100% filled

### PR DESCRIPTION
**Pull request by krzysiekwie**
old strings need review

### CODE REVIEW

**Comment by tessus** [comment]
I'm not sure what you did, but this PR is almost impossible to review. It shows diff on almost every line. A few items were translated which should not be. This will take a very long time to review.

**Comment by tessus** [comment]
please rebase the PR to the current git master from upstream. Maybe this helps.

**Comment by tessus** [comment]
You can have a look at how such a PR should look like:

https://github.com/laurent22/joplin/pull/2846/files
https://github.com/laurent22/joplin/pull/2839/files

and now look at yours:

https://github.com/laurent22/joplin/pull/2841/files

File: CliClient/locales/pl_PL.po [hunk 1]
```diff
@@ -44,13 +43,11 @@ msgstr "Nie wybrano notatnika."
 msgid "No notebook has been specified."
 msgstr "Notatnik nie został określony."
 
-#: /Users/tessus/data/work/joplin/Tools/../CliClient/app/app.js:136
-#: /Users/tessus/data/work/joplin/Tools/../CliClient/app/cli-utils.js:160
+#: /Users/tessus/data/work/joplin/Tools/../CliClient/app/app.js:136 /Users/tessus/data/work/joplin/Tools/../CliClient/app/cli-utils.js:160
```
[the remark this answers is not in the document]
Comment (krzysiekwie) [reply CliClient/locales/pl_PL.po]: Thanks for all the feedback, I'll do my best to learn from it.  I've used MemoQ to handle the .po file - usually I do localisations on Transifex or Smartling, or when I use MemoQ or SDL I don't handle integration part, this is literally my first time doing it on Github. Should I maybe try the Joplin forum for more advice/help?

[the remark this answers is not in the document]
Comment (krzysiekwie) [reply CliClient/locales/pl_PL.po]: I've re-downloaded the original .po file and filled in all empty strings using only po edit. Is there a way for me to check how it looks now on my own without taking your time?

[the remark this answers is not in the document]
Comment (tessus) [reply CliClient/locales/pl_PL.po]: You can do a `git diff`. But this only works, if you used git in the first place. You said you downloaded the file. If the diff only shows you the lines you really changed, all should be good.
Maybe the other programs messed up the file (as in changed or re-formatted non-changed text).

Don't worry, you can always create a PR.

Btw, if you need some help with git, I can give you a short explanation. Or github actually has a fantastic tool called [hub](https://hub.github.com/), which is basically a wrapper around git.

**Pull request closed without merging**